src/app/containers/RadioPageBlocks/Blocks/AudioPlayer/StructuredData.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import { string, number } from 'prop-types';
+
+const StructuredData = ({
```

Comment:
I might be inclined to rename this component to be 'AudioObjectStructuredData' (or something along those lines - let's get opinions from the other engineers)

Reply:
`StructuredData` here is only used within AudioPlayer. It's not to be used directly. AudioPlayer just uses it internally to render it's appropriate structured data. Since it's already in the AudioPlayer folder I think the long name is overkill.

Reply:
How about AudioObject instead of StructuredData? I just think that StructuredData is too generic